backend/.env

```
@@ -76,3 +76,7 @@ WEB_BACKLOG=2048
 # When running uvicorn locally (outside Docker), bind to IPv4 loopback to avoid
 # IPv6-only localhost resolution on some Linux distros.
 SERVER_HOST=127.0.0.1
+
+# Security
+SECURE_COOKIES=true
+BCRYPT_ROUNDS=12
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)